cli/src/cmd/forge/test.rs

```diff
@@ -438,29 +438,45 @@ fn short_test_result(name: &str, result: &TestResult) {
     let status = if result.success {
         Paint::green("[PASS]".to_string())
     } else {
-        let txt = match (&result.reason, &result.counterexample) {
-            (Some(ref reason), Some(ref counterexample)) => {
+        let txt = match (&result.reason, &result.counterexample, &result.counterexample_sequence) {
```

Comment:
nit: Can we unify these two? E.g. instead of `counterexample` and `counterexample_sequence` it could be `counterexamples` and we could change how we display based on the test type (fuzz vs invariant)

Reply:
i'm open to it, but it seems misleading no? It really is a counterexample, just of a different kind...

Reply:
Maybe an enum is a better fit? Something like:

```rs
enum Counterexample {
  Single(...),
  Sequence(...)
}
```